<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="videoShowWind" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>360</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVideoPlay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVideoStop">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtDevId">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设备ID:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtVideoStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>341</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labModelStatus">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>480</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>物模型状态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labPlayStatus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>播放状态:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetModel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>获取物模型</string>
   </property>
  </widget>
  <widget class="QLabel" name="labModelResp">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>480</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>物模型结果:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtModelStatus">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>281</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtModelResp">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>500</y>
     <width>301</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetModel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置物模型</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edtSetModelText">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>171</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labSetModelText">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>物模型设置内容:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtGetModelType">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="setModelType">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRecord">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始录像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnScreenShot">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>截图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTalk">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>静音</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
